FluentUI/src/main/java/com/microsoft/fluentui/contextualcommandbar/CommandItem.kt

```diff
@@ -0,0 +1,15 @@
+package com.microsoft.fluentui.contextualcommandbar
+
+import android.support.annotation.DrawableRes
+
+interface CommandItem {
```

Comment:
(also, I added a comment above but will add as it is most relevant here - should we have a property for a command string identifier?)